### PR TITLE
Wallet tests

### PR DESCRIPTION
**Pull request by dashuser33**
Add sequential tests for wallet methods.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,22 +19,6 @@ jobs:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
       - run: npm test
-  "node-latest":
```
Comment (jhodges10) [inline .circleci/config.yml]: Why did this get simplified so much? Is it because you use `buildQuick` on CircleCI?

**Comment by jhodges10** [comment]
Any idea why it's failing on `node-latest`?

File: .circleci/config.yml (hunk 1 above)
Comment (dashuser33) [reply .circleci/config.yml]: The test was previously being executed against node 8 and node latest. What is being proposed here is to only test on node-8 to reduce chance of tests interfering with each other. 

I will investigate the possibility of running these serially instead.

**Comment by dashuser33** [comment]
> Any idea why it's failing on node-latest?

It failed both before because the number of tries wasn't enough. Since I removed 'node - latest' after increasing the time to try, it didn't have the chance again.

**Comment by dashuser33** [comment]
Fixed the build so it will run on both node 8 and latest node, sequentially. Thanks for reviews. Going to merge.

**Pull request merged**